zarf-config.toml

```diff
@@ -6,6 +6,7 @@ agent_image_tag = 'local'
 
 # Tag for the zarf injector binary to use
 injector_version = '2025-03-24'
+injector_domain = 'zarf-init-resources.s3.us-east-1.amazonaws.com'
```

Comment:
When I execute `zarf dev inspect definition .` at the root of the repository - I get the following for the injector 
```yaml
- name: zarf-injector
  description: |
    Bootstraps a Kubernetes cluster by cloning a running pod in the cluster and hosting the registry image.
    Removed and destroyed after the Zarf Registry is self-hosting the registry image.
  required: true
  only:
    cluster:
      architecture: arm64
  files:
  - source: packages/zarf-registry/https:/zarf-init-resources.s3.us-east-1.amazonaws.com/injector/2025-03-24/zarf-injector-arm64
    shasum: f68cf097ace34bdd04f6b1571a8d8ab4e04a27614dd7e752199ff09e21f4089b
    target: "###ZARF_TEMP###/zarf-injector"
    executable: true
```

We need to determine why this is malformed.

Reply:
Looks like it's adding prepending the source path with the path to the imported package, without first checking if the file is a URL

Reply:
Ah because templating happens after imports, Zarf doesn't see the file as a URL during imports and therefore adds the path to the imported component